<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUmbralForm</class>
 <widget class="QDialog" name="QUmbralForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selección de umbrales</string>
  </property>
  <widget class="QScrollBar" name="horizontalScrollBarUM">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBarUm">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelUM">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Umbral Máximo</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUM_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Umbral Mínimo</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberUM">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberUm">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAceptar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancelar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalScrollBarUM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberUM</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBarUm</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberUm</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
